src/Ztm.Zcoin.Synchronization/AddressWatcher.cs

```diff
@@ -158,9 +158,8 @@ public class AddressWatcher : ITransactionConfirmationListener
                 }
 
                 // Add debit to address.
-                balances[address] = balances.ContainsKey(address)
-                    ? balances[address] + spend.Value
-                    : spend.Value;
+                Money val;
+                balances[address] = balances.TryGetValue(address, out val) ? val + spend.Value : spend.Value;
```

Comment:
We can do something like this `balances.TryGetValue(address, out var val)` and remove the above line.

Reply:
Good to know this sugar syntax